src/package/nuspec/TestPlatform.TestHost.nuspec

```diff
@@ -25,12 +25,6 @@
         <dependency id="Newtonsoft.Json" version="$JsonNetVersion$"/>
       </group>
 
-      <group targetFramework="uap10.0">
```

Comment:
I don't think it makes sense to have netstandard2.0 for testhost but then I am not sure what to ship for uap?

Reply:
Why it does not make sense to have netstandard2.0 for testhost?

Reply:
As we don't target netcoreapp, I thought this package is meant for specific scenario and so `netstandard2.0` seemed too generic.